tests/migrations/test_operations.py

```diff
@@ -2240,6 +2248,23 @@ def test_run_python_noop(self):
             operation.database_forwards("test_runpython", editor, project_state, new_state)
             operation.database_backwards("test_runpython", editor, new_state, project_state)
 
+    @skipUnlessDBFeature('supports_foreign_keys', 'can_introspect_foreign_keys')
+    def test_no_dbconstraint_added_during_field_change(self):
+        """
+        #28298
```

Comment:
As discussed in [Coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style), use a descriptive docstring and reserve ticket references for obscure issues. I think the expected behavior is straightforward enough that a ticket reference isn't needed.